roles/elastic_search/tasks/main.yaml

```diff
@@ -9,3 +9,5 @@
 #- include: 130_docker_python_libs.yaml
 
 - include: 150_elastic_search_docker_container.yaml
+
+# - include: 160_elastic_search_postconfig.yaml
```

Comment:
After moving from v2 to v5, Elasticsearch excluded from configuration shards and replicas count. Now you need to specify it via curl request for every separate shard. I left it for now before discussing the flow of the indexes creating. 
 - If we are going to define services indexes for elasticsearch during deployment - this tasks file will contain definition of those services
- if services will handle creating indexes by themselves - we will remove this task file.